tests/publiccloud/prepare_tools.pm

```diff
@@ -56,9 +58,7 @@ sub run {
     record_info('EC2', script_output('aws --version'));
 
     # Install Azure cli
-    zypper_call('-q in python3-devel');
     assert_script_run("pip3 install -q --ignore-installed azure-cli", 240);
-    assert_script_run("sed -i 's/^python /python3 /' /usr/bin/az");    # Workaround to force the use of python3, needed for performance!
```

Comment:
why is it not needed anymore?

Reply:
Yes! Like @asmorodskyi tries to do, but this method seems to work, link in /etc/alternatives was not.